scripts/bootstrap_cluster.sh

```diff
@@ -124,7 +129,7 @@ install_helm_chart_with_values() {
 
 
 install_argocd_helm_chart() {
-    helm install argocd argo/argo-cd --version 5.16.13 \
+    helm install argod argo/argo-cd --version 5.16.13 \
```

Comment:
we are directly installing argo/argo-cd, without using helm repo add .. , what if initially the machine do not have that chart